test/e2e-networking-library.sh

```diff
@@ -20,7 +20,7 @@ function install_istio() {
   fi
 
   # TODO: Figure out the commit of net-istio.yaml from net-istio.yaml
-  local NET_ISTIO_COMMIT=f64ed34d3776a444372483dddc15a330c6c1ac53
+  local NET_ISTIO_COMMIT=3104cca9922b1f0ec9c21d4413bbc9d8d3590cc9
```

Comment:
@JRBANCEL I was talking about this in my 1.5 comment.  I was under the impression that `f64ed34` was 1.4

Reply:
I see, yes `istio-stable` in `f64ed34` is 1.4.6.

Reply:
Unrelated to the kind bits, but that's probably why `pull-knative-serving-integration-tests` is failing :(